docs/embedded/tutorials/doc-processing-acs.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Document Processing with ACS
```

Comment:
Spell out ACS everywhere... it's not commonly used for Azure Cognitive Services, but also easily confused with other services (including the retired Azure Access Control Service).